project/AppDependencies.scala

```diff
@@ -3,7 +3,7 @@ import sbt._
 
 object AppDependencies {
 
-  val bootstrapVersion = "8.4.0"
+  val bootstrapVersion = "8.5.0"
 
   val compile = Seq(
     "uk.gov.hmrc"              %% "bootstrap-backend-play-30"      % bootstrapVersion,
```

Comment:
upgrade `internal-auth-client-play-30` to "2.0.0"
upgrade `scalatest` to 3.2.18
upgrade `scalatestplus-play` to 7.0.1
upgrade `mockito-core` to 5.11.0
upgrade `flexmark-all` to 0.64.8

Reply:
and add `play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientV2Module"` to application.conf